content/developer/howtos/standalone_owl_application.rst

```diff
@@ -0,0 +1,171 @@
+=====================================
+Creating a standalone Owl application
```

Comment:
title should probably be `Create a standalon Owl application` to make it look more similar to other howtos